src/Baballonia/Services/LogFileProvider.cs

```diff
@@ -18,9 +18,27 @@ public LogFileProvider()
             if (!Directory.Exists(Utils.UserAccessibleDataDirectory)) // Eat my ass windows
                 Directory.CreateDirectory(Utils.UserAccessibleDataDirectory);
 
-            var logPath = Path.Combine(Utils.UserAccessibleDataDirectory, "latest.log");
+            // Rotate log files: latest.log -> old.log -> older.log
```

Comment:
I don't know if we need to keep the latest three log files here. One is enough for our purposes.

Reply:
It wipes the log file on start, and it's pretty common to have a crash, restart the app, and when that doesn't work try to go to the log to figure out why and/or try to send it. Or another common scenario, "huh that was weird, but I want to just run the app and I'll investigate that bug later." So you restart the app and if so, it'll overwrite the log

Reply:
Hm. Instead of the latest 3 files, or a situation in which we risk overwriting, let's create a new log file every time the app starts. I know many apps are constructed like that.

Reply:
Check out my latest commit, I think the date-based system solves this in a better way than "old.log" and "older.log". Currently has a max of 10 log files and removes the oldest